### PR TITLE
Fix/ignored return value

### PR DESCRIPTION
**Pull request by LordCheta**

### CODE REVIEW

File: test/integration/PartnerRegistrar.spec.ts [hunk 1]
```diff
@@ -309,18 +309,6 @@ describe('New Domain Registration', () => {
...
-    const canReveal = await partnerProxyAsNameOwner.canReveal(commitment);
-
-    expect(canReveal).to.be.true;
-
-    await time.increase(1);
-
-    const canReveal = await partnerProxyAsNameOwner.canReveal(commitment);
-
-    expect(canReveal).to.be.true;
-
```
Comment (sergio-jara) [inline test/integration/PartnerRegistrar.spec.ts:323]: why removed? 👀

**Review by sergio-jara — CHANGES_REQUESTED** [review]
we need to add the unit tests that covers the code changes

File: test/integration/PartnerRegistrar.spec.ts (hunk 1 above)
Comment (LordCheta) [reply test/integration/PartnerRegistrar.spec.ts:323]: That's a mistake, I was supposed to delete the duplicate and then correct the mistake on line 320. I've fixed it now, thanks for spotting it

**Review by sergio-jara — APPROVED** [review]
awesome work

**Pull request merged**